effectful/src/Effectful/Console/Static.hs

```diff
@@ -0,0 +1,27 @@
+-- | An effect for reading/writing to standard streams.
+-- This module does intentionally provide no operations for the effect; Instead
+-- those are implemented in more specfic module like:
+--
+--  * "Effectful.ByteString.Static"
+--  * "Effectful.ByteString.Lazy.Static"
+--
+module Effectful.Console.Static
```

Comment:
No `Console` effect please. I'm not convinced it's a good idea, especially since log-effectful already exists.

Reply:
Feel free to move the effect and associated functions to `effectful-console` :+1:

Reply:
Ok, thanks! I removed the `Console` effect and the associated functions.

> You can put it in the org, assuming that you're interested in maintaining it, because it looks like the org is getting more repos that just sit there with no release.

Yes, I know; I also did not find the time to release typed-process-effectful properly :grimacing:

Reply:
Actually, I guess it's fine, you can keep it here. But please separate Console and FileSystem functions.

Reply:
Eh, looks like I was too late with this comment :disappointed: Luckily you didn't force push, so feel free to revert.

Reply:
Ok, nice; I'll do that!